### PR TITLE
CAF r1 fixes

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/cloud-adoption/appendix/azure-scaffold.md [hunk 1]
```diff
@@ -174,14 +175,14 @@ Planning and preparing for your identity and access controls and following Azure
...
 
 ### Azure Security Center
 
 The [Azure Security Center](/azure/security-center/security-center-intro) provides a unified view of the security status of resources across your environment in addition to advanced threat protection. Azure Security Center is an open platform that enables Microsoft partners to create software that plugs into and enhance its capabilities. The baseline capabilities of Azure Security Center (free tier) provides assessment and recommendations that will enhance your security posture. Its paid tiers enable additional and valuable capabilities such as Just In Time admin access and adaptive application controls (whitelisting).
 
 > [!TIP]
->Azure security center is a very powerful tool that is constantly being enhanced and incorporating new capabilities you can leverage to detect threats and protect your enterprise. It is highly recommended to always enable ASC.
+>Azure Security Center is a very powerful tool that is constantly being enhanced and incorporating new capabilities you can use to detect threats and protect your enterprise. It is highly recommended to always enable Azure Security Center.
```
Comment (MikeWasson) [inline docs/cloud-adoption/appendix/azure-scaffold.md]: nit: Could remove 'very' here

File: docs/cloud-adoption/decision-guides/encryption/overview.md [hunk 2]
```diff
@@ -33,24 +33,25 @@ When planning a cloud migration, the following table describes how you can store
...
-Cloud native key management assumptions: Using a cloud native key management system assumes the following:
+Using a cloud native key management system assumes the following:
 
 - You trust the cloud key management solution with creating, managing, and hosting your organization's secrets and keys.
 - You enable all on-premises applications and services that rely on accessing encryption services or secrets to access the cloud key management system.
 
 ### Hybrid (bring your own key)
 
-With a bring-your-own-key approach, you generate keys on dedicated HSM hardware within your on-premises environment, then transfer the keys to a secure cloud key management system for use with cloud resources.
+With a bring your own key approach, you generate keys on dedicated HSM hardware within your on-premises environment, then transfer the keys to a secure cloud key vault for use with cloud resources.
```
Comment (MikeWasson) [inline docs/cloud-adoption/decision-guides/encryption/overview.md]: I think the hyphens are OK here "bring your own key" modifies "approach".  It's a bit awkward, we could just say "with this approach"

File: docs/cloud-adoption/appendix/azure-scaffold.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: Fixed

File: docs/cloud-adoption/decision-guides/encryption/overview.md (hunk 2 above)
Comment (alexbuckgit) [reply docs/cloud-adoption/decision-guides/encryption/overview.md]: Fixed

**Pull request merged**